Department-for-Transport-Disruptions/create-disruptions-data
# Update ParticipantRef field with org name in siri sx xml

Testing instructions

- Create a new disruption or update an existing disruption. After the validator lambda has completed and the data is in Siri SX XMl, access this data using the API and verify that the org name is added in the ParticipationRef field
